dashboard/app/controllers/regional_partners_controller.rb

```diff
@@ -146,6 +146,9 @@ def replace_mappings
     redirect_to @regional_partner
   end
 
+  def regional_partner_search
```

Comment:
Just for my own understanding, what is the purpose of this blank `regional_partner_search` method?

Reply:
I don't think they're needed but I personally think it's a bit clearer what routes direct to this controller when there are empty methods. Happy to remove if it's more confusing than it's worth though!

Reply:
(I genuinely thought they were necessary until I read [this documentation](https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-by-default-convention-over-configuration-in-action))

Reply:
Oh I see! Thank you!